template/native/render_test.go

```diff
@@ -8,11 +8,10 @@ import (
 	"io/ioutil"
 	"testing"
 
-	"github.com/go-vela/types/raw"
+	goyaml "github.com/goccy/go-yaml"
```

Comment:
Do you think we should keep the import for `goccy/go-yaml` consistent?

Earlier in this PR we named it `yml`:

https://github.com/go-vela/compiler/blob/4c89c8798a5381b222e73b6d7b04d5d9e5b8e343/compiler/native/compile.go#L17

But then here, we named it `goyaml`.

I'm not proposing one over the other, but more so that we should choose one.

Reply:
I have zero preference. What would you like?

Reply:
Honestly, same haha 😅 

Since there is no preference, maybe we choose `yml` because it's shorter?

Reply:
Awesome! will update